certs.go

```diff
@@ -0,0 +1,46 @@
+package certs
+
+// Cert is a certificate name.
+type Cert string
+
+func (c Cert) String() string {
+	return string(c)
+}
+
+// These constants used as Cert parsing a secret received from the API.
+const (
+	APICert                Cert = "api"
+	AppOperatorAPICert     Cert = "app-operator-api"
+	AWSOperatorAPICert     Cert = "aws-operator-api"
+	CalicoEtcdClientCert   Cert = "calico-etcd-client"
+	ClusterOperatorAPICert Cert = "cluster-operator-api"
+	EtcdCert               Cert = "etcd"
+	Etcd1Cert              Cert = "etcd1"
+	Etcd2Cert              Cert = "etcd2"
+	Etcd3Cert              Cert = "etcd3"
```

Comment:
I added these.

Reply:
This looks really unfortunate for me - why can `EtcdCert` not be a slice of certs? So it can either have length 1 or 3?

Now you end up with 4 certs in `AllCerts` where one is unused?

Reply:
Thinking about it more - why even have these magic strings here instead of the provider operator? We use the key functions for these kinds of things in the provider operators anyway. Why are we making the distinction here to have them in a library?

Additionally by getting rid of the `cluster` parts of the lib, it makes sense to pull this stuff to the provider operators as well.

Reply:
We talked. The components here act as identifiers for certificates we generate. They are used as label selector. I tried to improve the comments.